src/lavinmq/http/controller.cr

```diff
@@ -142,14 +142,13 @@ module LavinMQ
       end
 
       private def parse_body(context) : JSON::Any
```

Comment:
Or should this return `JSON::Any?`  so return nil if no body or content_length == 0?